x/autopilot/keeper/airdrop.go

```diff
@@ -0,0 +1,46 @@
+package keeper
+
+import (
+	"errors"
+	"fmt"
+
+	errorsmod "cosmossdk.io/errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	transfertypes "github.com/cosmos/ibc-go/v5/modules/apps/transfer/types"
+	channeltypes "github.com/cosmos/ibc-go/v5/modules/core/04-channel/types"
+
+	"github.com/Stride-Labs/stride/v7/utils"
+	"github.com/Stride-Labs/stride/v7/x/autopilot/types"
+)
+
+func (k Keeper) TryUpdateAirdropClaim(
+	ctx sdk.Context,
+	packet channeltypes.Packet,
```

Comment:
I don't think this packet arg is used - but I'm wondering if there's anything else we should check in the packet

wdyt?